app/controllers/events/view/export.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+
+const { Controller, run, $ } = Ember;
+export default Controller.extend({
+  eventExportStatus : '',
+  eventDownloadUrl  : '',
+  actions           : {
+    startGeneration() {
+      let payload = {
+        image    : this.get('model.exportData.dataImage'),
+        video    : this.get('model.exportData.dataVideo'),
+        audio    : this.get('model.exportData.dataAudio'),
+        document : this.get('model.exportData.dataDocument')
+      };
+      let exportUrl = `/events/${this.get('model').currentEventID}/export/json`;
+      this.get('loader')
+        .post(exportUrl, payload)
+        .then(data => {
+          let download_url = data.task_url.split('/').splice(2, 4).join('/');
```

Comment:
Its never advisable to play with the url like this. Split and splice and join is more error prone now than the previous sub string implementation.